### DISCUSSION TITLE
ChatGPT

### DISCUSSION THREAD

**Discussion by anonimusprogramus** [feathersjs/feathers#2924] [General]
Stumble upon this :smile: 

![2022-12-08 19-09-59](https://user-images.githubusercontent.com/7464749/207177235-ca91dc1a-d488-4267-ac9b-46ed2fafedf5.png)

**Comment by daffl** [comment]
There was a bit of discussion about this [on Discord](https://discord.gg/qa8kez8QBx) as well. It's pretty good but unfortunately won't have any of the new things that are about to be officially released soon.

**Reply by anonimusprogramus** [reply]
oh thanks for the link, didn't know that :+1:

**Comment by jimoquinn** [comment]
Having never used typebox, I was sorting the syntax for arrays of objects and nested objects, so I could port over my resume.json schema from Feathers v4 to v5.  

After futzing for an hour or so with a half dozen experiments, I asked chatgpt v3 to "generate a typebox schema using the full resume.json" and low and behold, it produced code that was about 95% correct!  A day later I'm still kinda blown away.

Also asked chatgpt v3 to clean up and formalize a couple of the new documentation pages thinking it would render parts of it useless, but much to my surprise it left the bulk of the text alone but produced little improvements all throughout.
